example-ClientScript.ts

```diff
@@ -3,32 +3,44 @@
  * @NScriptType ClientScript
  */
 
-import {EntryPoints} from 'N/types'
+import { EntryPoints } from 'N/types';
 import search = require('N/search');
 
 export function pageInit(ctx: EntryPoints.Client.pageInitContext) {
-  if (ctx.mode != 'edit') return;
+    if (ctx.mode != 'edit') return;
 
-  const customerId = ctx.currentRecord.getValue('entity'); // Assume this script is running on a transaction
-  search.lookupFields.promise({ type: 'customer', id: customerId, columns: ['companyname', 'datecreated', 'entitystatus'] }).then((values) => {
-    const name   = values.companyname as string;
-    const date   = values.datecreated as string;
-    const status = values.entitystatus as { value: string, text: string }[];
-    console.log('Customer', name, 'created at', date, 'status', status);
-  });
+    const customerId = ctx.currentRecord.getValue('entity'); // Assume this script is running on a transaction
+    search.lookupFields
```

Comment:
@accountingnerd Do you know if there is a way to set it not to break out the line like this?  We prefer the way it was before, on a single line

Reply:
Sure. If you include a comment `// prettier-ignore` in the file, prettier will ignore the next node in the AST.

I'm pushing up a commit right now with that on this file so you can see.